common/src/main/kotlin/common/Diagnostics.kt

```diff
@@ -0,0 +1,25 @@
+package common
+
+enum class DiagnosticSeverity {
+    Error, Warning, Info
```

Comment:
Enum values in Java and Kotlin are typically CAPS_LOCK

https://kotlinlang.org/docs/enum-classes.html

Reply:
According to the latest [Kotlin coding conventions](https://kotlinlang.org/docs/coding-conventions.html), both are fine:

> For enum constants, it's OK to use either uppercase underscore-separated names ([screaming snake case](https://en.wikipedia.org/wiki/Snake_case)) (enum class Color { RED, GREEN }) or upper camel case names, depending on the usage.

I personally prefer `PascalCase`, hence why it's used here